projects/sudoers/fuzz_sudoers_parse.c

```diff
@@ -39,26 +39,27 @@ limitations under the License.
 #include "cvtsudoers.h"
 #include <gram.h>
 
-
 int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
+    if (size < 5) {
+        return 0;
+    }
+
     char filename[256];
-    sprintf(filename, "/tmp/libfuzzer.%d", getpid());
+    sprintf(filename, "/tmp/fuzz-sudoers.XXXXXX");
 
-    FILE *fp = fopen(filename, "wb");
-    if (!fp) {
+    int fp = mkstemp(filename);
+    if (fp < 0) {
         return 0;
     }
-    fwrite(data, size, 1, fp);
-    fclose(fp);
+    write(fp, data, size);
+    close(fp);
 
     // main entry point for the fuzzer
-    fp = fopen(filename, "rb");
-    init_parser(filename, false, true);
-    sudoers_parse_ldif(&parsed_policy, fp, NULL, true);
-
-    //fclose(fp);
-    unlink(filename);
-
+    FILE *fd = fopen(filename, "rb");
+    if (fd != NULL) {
+        init_parser(filename, false, true);
+        sudoers_parse_ldif(&parsed_policy, fd, NULL, true);
+    }
+    remove(filename);
```

Comment:
missing fclose.

Reply:
No missing - `sudoers_parse_ldif` closes the file itself: https://github.com/sudo-project/sudo/blob/aa50aaf8dae9c3c5e6c27ee743cedec1b0524fba/plugins/sudoers/parse_ldif.c#L777